package.json

```diff
@@ -74,7 +74,7 @@
         "mini-css-extract-plugin": "^0.8.0",
         "mock-local-storage": "^1.1.15",
         "mojito-rb-gen": "^0.0.1",
-        "node-sass": "4.13.1",
+        "sass": "1.34.1",
```

Comment:
I think the dependencies are alphabetized, can we move this down if they are?

Reply:
💀 yep yep on it!